docs/changes.txt

```diff
@@ -1,11 +1,23 @@
 * changes v5.1.0 -> v5.x.x
 
+##
+## Algorithms
+##
+
+- Added hash-mode 18400 (Open Document Format (ODF) 1.2 (SHA-256, AES))
+
 ##
 ## Bugs
 ##
 
 - Fixed output of IKE PSK (mode 5300 and 5400) hashes to have separators at right position
 
+##
+## Technical
+##
+
+- Tests: Added hash-mode 18400 (Open Document Format (ODF) 1.2 (SHA-256, AES))
```

Comment:
minor consideration: we normally do not add any lines/entries for new tests in docs/changes.txt (except if the tests were missing in older releases and we finally managed to complete/add the tests). I think it is enough to just have the other line under Algorithms, because usually if we add a new algo it comes accompanied with tests (as in this case). Again, this is not really a problem. It's just a note and a hint to make it more consistent for other/future PRs ;)